package/ios/Frame Processor/FrameProcessorPlugin.h

```diff
@@ -26,3 +26,30 @@
 - (id _Nullable)callback:(Frame* _Nonnull)frame withArguments:(NSDictionary* _Nullable)arguments;
 
 @end
+
+#define VISION_CONCAT2(A, B) A##B
+#define VISION_CONCAT(A, B) VISION_CONCAT2(A, B)
+
+#define VISION_EXPORT_FRAME_PROCESSOR(frame_processor_class, frame_processor_plugin_name)                                                  \
+  +(void)load {                                                                                                                            \
+    [FrameProcessorPluginRegistry addFrameProcessorPlugin:@ #frame_processor_plugin_name                                                   \
+                                          withInitializer:^FrameProcessorPlugin*(NSDictionary * options) {                                 \
+                                            return [[frame_processor_class alloc] init];                                                   \
```

Comment:
Here as well! `options` are not passed.

Maybe we should always have `init`/constructors with options, but we can just make them nullable?

Reply:
Good catch, let me fix it

Reply:
I added default `- initWithOptions` constructor to the `FrameProcessorPlugin` class, so users may override it to consume that `options` object. Also it seems that object wasn't marked in TypeScript, so I added it and adjusted rest of iOS/Android code to consume it properly. Let me know if it's ok